SCAutolib/cli_commands.py

```diff
@@ -165,7 +201,102 @@ def cleanup(ctx):
     exit(ReturnCode.SUCCESS.value)
 
 
-cli.add_command(setup_ca)
-cli.add_command(prepare)
-cli.add_command(setup_user)
-cli.add_command(cleanup)
+@cli.group(cls=NaturalOrderGroup, chain=True)
+@click.pass_context
+def gui(ctx):
+    """ Run GUI Test commands """
+    pass
+
+
+@gui.command()
+def init():
+    """ Initialize GUI for testing """
+    return "init"
+
+
+@gui.command()
+@click.argument("name")
+def assert_text(name):
+    """ Check if a word is found on the screen """
+    return f"assert_text:{name}"
+
+
+@gui.command()
+@click.argument("name")
+def assert_no_text(name):
+    """ Check that a word is not found on the screen """
+    return f"assert_no_text:{name}"
+
+
+@gui.command()
+@click.argument("name")
+def click_on(name):
+    """ Click on object containing word """
+    return f"click_on:{name}"
+
+
+@gui.command()
+def check_home_screen():
+    """ Check if screen appears to be the home screen """
+    return f"check_home_screen"
+
+
+@gui.command()
+@click.argument("keys")
+def kb_send(keys):
+    """ Send key(s) to keyboard """
+    return f"kb_send:{keys}"
+
+
+@gui.command()
+@click.argument("keys")
+def kb_write(keys):
+    """ Send string to keyboard """
+    return f"kb_write:{keys}"
+
+
+@gui.command()
+def done():
+    """ cleanup after testing """
+    return "done"
+
+
+@gui.result_callback()
+@click.pass_context
+def run_all(ctx, actions):
+    click.echo(actions)
+
+    from SCAutolib.models.gui import GUI
+    gui = GUI()
+    for action in actions:
+        if "init" in action:
+            gui.__enter__()
+        if "assert_text" in action:
+            assert_text = action.split(":")[1]
```

Comment:
maybe the delimiter is not good in that it is really common and we may need to send it, but maybe not, I need to search it more.

Reply:
What about using split()'s maxsplit option to only split on the first instance like this:
```
>>> action = "assert_text:I need to find this:test"
>>> action.split(':',1)[1]
'I need to find this:test'
```

That should allow us to use the ':' to pass `<action>:<arg>` even if `<arg>` includes a ':'.

Reply:
Yeah, that can actually work well since we have a standardized first text that we know it doesn't have ':' in it.